voorhoede/head-start
# chore: import path aliases

# Changes

- Configure and use path aliases to improve readability. So for example `import Button from '../../../components/Button/Button.astro' is simplified to '@components/Button/Button.astro' no matter on which directory level it's used.
- Change astro.config.mjs to .ts so the same coding style applies to it.

# Associated issue

N/A

# How to test

- Open preview link. Verify pages still render.
- Use an `@block/component/layout/lib` import statement somewhere if you want to try it during development.

# Checklist

- ~~I have added/updated tests to prove that my feature works (if not possible please explain why)~~
- ~~I have made changes to the README and if the change affects the project setup (npm commands changed, new service added, environmental variable added)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
